puppet/modules/jenkins_job_builder/files/centos.org/ansible/fetch_debug_files.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  tasks:
+    - name: 'Find any files from debug'
+      find:
+        paths: '/tmp'
+        recurse: yes
+      register: debug_files
+    - name: 'Fetch debug files'
+      fetch:
+        src: "{{ item.path }}"
+        dest: "/home/foreman/workspace/{{ job_name }}/debug"
```

Comment:
I think the workspace should be a parameter.

Reply:
I think you're right it's probably in the environment variables.

Reply:
Or for this instance, you can pass it directly when calling the playbook as a variable

Reply:
Oh, wait do you mean the whole thing should be a variable?

Reply:
I think `/home/foreman/workspace` should be a variable here.